src/mgr/ActivePyModules.cc

```diff
@@ -1029,9 +1029,9 @@ PyObject* ActivePyModules::get_perf_schema_python(
       with_gil(no_gil, [&, key=ceph::to_string(key), state=state] {
         f.open_object_section(key.c_str());
         for (auto ctr_inst_iter : state->perf_counters.instances) {
-          const auto &counter_name = ctr_inst_iter.first;
-          f.open_object_section(counter_name.c_str());
+          const string &counter_name = ctr_inst_iter.first;
```

Comment:
I think we can leave it there as I found out that creating the section with c_str excluded labels because of null characters. I mean, it is obviously unnecessary here, but I thought it would fix future issues with labels by using string directly.

Reply:
> I thought it would fix future issues with labels by using string directly.

>           f.open_object_section(counter_name);

Using a string with NUL chars in it as a JSON key is unlikely to lead to anything good...

Reply:
> Using a string with NUL chars in it as a JSON key is unlikely to lead to anything good...

Sorry, I should have been more explicit: please drop this leftover change as it doesn't address any potential future issue.